benchmarks/synchrobench/src/main/java/com/yahoo/oak/OakBenchHash.java

```diff
@@ -62,12 +62,15 @@ public void close() {
     /** {@inheritDoc} **/
     @Override
     public boolean ascendOak(BenchKey from, int length, Blackhole blackhole) {
-        throw new UnsupportedOperationException("ALL ITERATORS ARE NOT YET SUPPORTED FOR HASH");
+
+        boolean result = createAndScanView(oakHash, length, blackhole);
+
+        return result;
     }
 
     /** {@inheritDoc} **/
     @Override
     public boolean descendOak(BenchKey from, int length, Blackhole blackhole) {
-        throw new UnsupportedOperationException("ALL ITERATORS ARE NOT YET SUPPORTED FOR HASH");
+        throw new UnsupportedOperationException("Descending iterator are not defined for Hash");
```

Comment:
In `JavaHashMap` the `ascendOak` is invoked when `descendOak` is requested. I think it is reasonable. Why not to implement it the same way here?

Reply:
I think it's a bit misleading. But for the sake of unification, we can make it the same as in JavaHashMap

Reply:
👍